scheduler/src/cook/mesos/monitor.clj

```diff
@@ -21,8 +21,7 @@
             [clojure.tools.logging :as log]
             [cook.mesos.share :as share]
             [cook.mesos.util :as util]
-            [datomic.api :as d :refer (q)]
-            [metatransaction.core :refer (db)]
+            [datomic.api :as d :refer (db, q)]
```

Comment:
Nitpick, please don't refer db since it is overloaded

Reply:
Done, thanks